docs/components/Checkbox/Mobile.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Checkbox, CheckboxGroup } from "@jobber/components-native";
+
+export default {
+  title: "Components/Selections/Checkbox/Mobile",
+  component: Checkbox,
+  subcomponents: { CheckboxGroup },
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
+    viewport: { defaultViewport: "mobile1" },
+  },
+} as ComponentMeta<typeof Checkbox>;
+
+const BasicTemplate: ComponentStory<typeof Checkbox> = args => (
+  <Checkbox {...args} />
+);
+
+export const Basic = BasicTemplate.bind({});
+Basic.args = {
+  label: "Check me out",
+  name: "storyCheckbox",
+};
+
+const ControlledTemplate: ComponentStory<typeof Checkbox> = args => {
+  const [checked, setChecked] = useState(false);
+  return (
+    <Checkbox
+      {...args}
+      checked={checked}
+      onChange={() => setChecked(!checked)}
+    />
+  );
+};
+
+export const Controlled = ControlledTemplate.bind({});
+Controlled.args = {
+  label: "Check me out",
+};
+
+export const Indeterminate = BasicTemplate.bind({});
```

Comment:
I think it may be worth adding more information/context around this example in the docs section explaining when it would be used and linking to this example. Mobile storybook has the context for this so it's probably worth porting over.

Reply:
Good point! I added this section that we currently have on mobile to the updated docs, it applies for both Web and Mobile components. I also added a section for mobile's CheckboxGroup